docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+# Command-Line Tutorial
+
+This tutorial will explain how to use the HELM command line tools to run benchmarks, aggregate statistics, and visualize results.
+
+In this tutorial, we will run two runs using the `mmlu` scenario on the `huggingface/gpt-2` model. The `mmlu` scenario implements the **Massive Multitask Language (MMLU)** benchmark from [this paper](https://arxiv.org/pdf/2009.03300.pdf), and consists of a Question Answering (QA) task using a dataset with questions from 57 subjects such as elementary mathematics, US history, computer science, law, and more. We will run two runs: the first using questions about anatomy, and the second using questions about philosophy.
+
+## Using `helm-run`
+
+`helm-run` is a command line tool for running benchmarks.
+
+To run this benchmark using the HELM command-line tools, we need to specify **run spec descriptions** that describes the desired runs. For this example, the run spec descriptions are `mmlu:subject=anatomy,model=huggingface/gpt-2` (for anatomy) and `mmlu:subject=philosophy,model=huggingface/gpt-2` (for philosophy).
+
+Next, we need to create a **run spec configuration file** contining these run spec descriptions. A run spec configuration file is a text file containing `RunEntries` serialized to JSON, where each entry in `RunEntries` contains a run spec description. The `description` field of each entry should be a **run spec description**. Create a text file named `run_spec.conf` with the following contents:
+
+```
+entries: [
+  {description: "mmlu:subject=anatomy,model=huggingface_gpt-2", priority: 1},
+  {description: "mmlu:subject=philosophy,model=huggingface_gpt-2", priority: 1},
+]
+```
+
+We will now use `helm-run` to execute the runs that have been specified in this run spec configuration file. Run this command:
+
+```
+helm-run --run-specs run_spec.conf --local --suite v1 --max-eval-instances 10
+```
+
+The meaning of the additional arguments are as follows:
+
+- `--local` specifies not to use the Stanford CRFM API. You should always use `--local` unless you are a Stanford CRFM API user.
+- `--suite` specifies a subdirectory under the output directory in which all the output will be placed.
+- `--max-eval-instances` limits evaluation to only the first *N* inputs (i.e. instances) from the benchmark.
+
+`helm-run` creates an environment directory environment and an output directory by default.
+
+-  The environment directory is `prod_env/` by default and can be set using `--local-path`.
+-  The output directory is `benchmarking_output/` by default and can be set using `--output-path`.
+
+After running this command, navigate to the `benchmarking_output/runs/v1/` directory. This should contain a two sub-directories named `mmlu:subject=anatomy,model=huggingface_gpt-2` and `mmlu:subject=philosophy,model=huggingface_gpt-2`. Note that the names of these sub-directories is based on the run spec descriptions we used earlier, but with `/` replaced with `_`.
```

Comment:
Before we also had `/` replaced with `_`...  I think maybe we want to point people to `run_specs.py` where `mmlu:subject=...` is defined and `run_expanders.py` where the other stuff is defined.  I don't think we want to make this tutorial too long because we should explain things properly in a separate file, but we should perhaps have a few more pointers.